agenda/presentation/src/main/java/com/nibalk/tasky/agenda/presentation/home/HomeViewModel.kt

```diff
@@ -30,11 +39,10 @@ class HomeViewModel(
     init {
         viewModelScope.launch {
             setProfileInitials()
-//            snapshotFlow { state.selectedDate }
-//                .distinctUntilChanged()
-//                .collectLatest {
-//                    getAgendaItems()
-//                }
+            getAgendaItemsFromLocal()
+        }
+        viewModelScope.launch {
+            getAgendaItemsFromRemote(true)
```

Comment:
I have no idea what this `true` stands for, I'd therefore recommend to always use named parameters for such unclear arguments 😄